app/presenters/atom_presenter.rb

```diff
@@ -0,0 +1,22 @@
+class AtomPresenter
+
+  def initialize(finder)
+    @finder = finder
+  end
+
+  def title
+    finder.name
+  end
+
+  def entries
+    @finder.results.documents
```

Comment:
Ah yeah. Didn't mean to leave that when I copied it from the view. Updated the commit.